.travis.yml

```diff
@@ -6,7 +6,13 @@ os:
 notifications:
   email: false
   slack:
-    secure: CcTUqFnp4f7ks1npvzDfODDuJ4xnTvTYSAzIXrKBo11zB4qhPwnQmoXsmas18jQoGMg/L4yyWnIpQykXolMfs1NeGJsqfERQz+Z7sAi/GAHU9licI5kK7AWpFeV56kKzQHNEes2GgSdLsLmIxHsd+WoCSC0OuJoYiEhw36GdhJ1hS7O4viy3QBcSIAVQB9Qxx45M8WpuAdtOj3sR4ceRHk0XRu2oeIlO8398eTuloYODTdZy3wKhmHSB34ur9OAdEg734Qsl6ws2lB1qUuqDpCtIsf4Bgo8QfGBgc/EVsh52HhwKJm87eDDIxDO5zS/HFs3tXpID8Jgc9jWZCUJkj7cO0gg04lXlLeQcUXYIwa1EkiSUg+wpgrAphWpvYHAQ9pZ26ugrP3G7IzWFm5tVKOZKLNDmQ8ivgVojYmK3aS6gOnHrhyq5HN6n4VEZqNAPy1AfObPCkqMSNIgPI0lGKPJMiJUzkHwLRbCinCiETnSvwQjLiEKPouE90EFzsw451hiKI9R40QXj2ig6PVkvglOVcNLq4J9Vt+P5c4dNSN6JOZS4IJv/loYZrqaxzaRvkG+8iizH00QHSizMbp71xYHp2IQHdXzZmf4tvHaaHWW0nxxHbd6XdYsPAQoVDCVk+MFTMBiTPnN4NG6uxXllqxhJrK21DesbsvzIhtS0GZs=
+    rooms:
+      - secure: fcmuyLTnHaC/KQA4KwJKJTuk7pk0/gbNcOJJ8p1lfLFt59ZSBezOKbatiJU6XuWH0y6zdKum8qnEIFfD5Iuj0PTKwzt1Miaep3w73kSo8UDxPQcb/TwNt5S+MOinPZrX21ZcTZ0N3iSB4l1RsgB6QlX6WkJd8sH2du4bjSFElzYz5P94G5qHVaotqWc6dhzbVZsQhUTFSEuzCQ1hxAWTJFSuJgJO7cqfxIYo0BWfiW1MOmIlKxbVxbhRKscQR4mWEFtW7eIkbCpk3zMPjnwIG1bOpMN5jbZXSGwWnK8GUUVwptf9NtRgZkC3B+ZVIwM69qkEZ4Pl/TUSwKQLulULeyQVb/OvZqPcCYuuSQye98620eJYPyqlHByYt35ImVI4hp/D3lB95lr29Ecj80A5RIrtgAkt81Id9cuxccuw8Ytu7+bMoaeRssTDWBLo/vDf0fFSgL1r/vG5RaabYcbPdcolsr2yG/3OLuW3sfx4FfuUigS8OMoRDWa63+aNS5mDo+5pWcXNoY4fJPUo1kTBnXXzWuqyFNYxqoljCFlE/o7zSsdJAzuS4iFhpNurZhcN7zEDQ8E1HqDuItuf595h9i++4lZN7JcRr83SRQxEse9HK+SuDxOG9Rqv6QDKIEFoWtUxgntmEumoh50s9NAojNPKkYI0IIXDwojaDt9mxlA=
```

Comment:
Seems like the rest of this file is using travis env variables instead of hashed secrets? Maybe we should keep this one the same?

Reply:
Yeah we should probably be consistent, unless we can't use travis variables @kozr